src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -346,6 +362,19 @@ bool PerformInteractionRequest::ProcessVRResponse(
     return false;
   }
 
+  if (FirstAnsweredInterface::VR == first_responser_) {
+    // After VR.PerformInteraction response HMI should close UI popup window
+    // if UI.PerformInteraction response comes
+    // after VR.PerformInteraction response.
+    // In this case SDL should send UI_ClosePopUp request.
+    const std::string kUIPerformInteractionMethodName = "UI.PerformInteraction";
```

Comment:
@YarikMamykin wrong name for `local` variable. Rename please to `method_name`

Reply:
@AByzhynar fixed in https://github.com/smartdevicelink/sdl_core/pull/3122/commits/8b1b9a79752b6e208df76048cff2e8c307b71492